### PR TITLE
Positive sign up tests

### PR DESCRIPTION
**Pull request by AhnessP**

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+using System;
```
Comment (eugenek7) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs]: Please, delete useless using

**Review by eugenek7 — APPROVED** [review]

File: EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            signUpPage = GetSignUpPage();
+            signUpPage.SendKeysToInputName(dataModel.NameForSignUp);
+            signUpPage.SendKeysToInputEmail(dataModel.EmailForSignUp);
+            signUpPage.SendKeysToInputPhoneNumber(dataModel.LettersInPhoneNumber);
+            signUpPage.SendKeysToInputPassword(dataModel.PasswordForSignUp);
+            signUpPage.SendKeysToInputConfirmPassword(dataModel.PasswordForSignUp);
+            signUpPage.ClickCreateAccountButton();
+            var actual = DatabaseManager.SendQuery(queryDataModel.SelectUserEmailByEmail, dataModel.EmailForSignUp);
+            var expected = dataModel.EmailForSignUp;
+            Assert.AreEqual(expected, actual,"User is unable to sign in with invalid phone number");
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs]: "User is unable to sign **up** with invalid phone number"

**Review by dmatsev — APPROVED** [review]

File: EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs (hunk 1 above)
Comment (AhnessP) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs]: fixed

File: EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs (hunk 2 above)
Comment (AhnessP) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckPositiveSignUpTests.cs]: fixed

**Pull request merged**